cves/kernel/CVE-2014-3183.yml

```diff
@@ -106,7 +120,8 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 0e40d35637d68f654b66f4562c9a914be7d06bd1
-  note: Discovered automatically by archeogit.
+  note: Discovered automatically by archeogit. Manually confirmed.
```

Comment:
I would suggest to add what was added and deleted within this commit for more clarity, but this is not a requirement just a suggestion.

Reply:
I added a description of what the patch did.